otterdog/eclipse-set.jsonnet

```diff
@@ -44,6 +44,14 @@ orgs.newOrg('eclipse-set') {
       delete_branch_on_merge: true,
       has_wiki: false,
       web_commit_signoff_required: false,
+      branch_protection_rules: [
+        orgs.newBranchProtectionRule('main') {
+          requires_pull_request: true,
```

Comment:
looks much better now.

The 3 settings:

- requires_pull_request
- allows_deletions
- allows_force_pushes

could be removed, as the same values are already inherited from the default config.

Reply:
ok, hope everythings ok this time :)